src/Forms/Checkbox.js

```diff
@@ -119,6 +125,8 @@ Please ensure you are either using a visible \`FormLabel\` or an \`aria-label\`
             );
         }
     },
+    /** Additional classes to apply to children */
+    childrenClassName: PropTypes.string,
```

Comment:
I think we have to be a little more careful with the name here - if the child is not a string, it won't add this class. Do we want that? Do we not? I don't know...

Reply:
I also thought perhaps this was not the best name. Would you have a suggestion of a more semantic, or explicit name? Perhaps `stringChildrenClassName`?

Reply:
I think `textClassName` would be fine